week1/homework.md

```diff
@@ -0,0 +1,59 @@
+# Homework week 1
+
+1. Go to `https://github.com/HackYourFuture-CPH/hyf-homework` and fork this repository. Now you should have the `hyf-homework` repository in your own github account.
+2. Clone the `hyf-homework` repository from YOUR github account.
+3. Inside the folder `git/week1` create a new file named `my_favorite_food.txt`, inside the file write your favorite food recipe, and commit your changes (you can just find a random recipe on google and paste it in the file :wink: ).
+4. Again inside the folder `git/week1` add a new file named `my_second_favorite_food.txt`, inside the file write the recipe for your second favorite food, and commit your changes.
+5. Inside the folder `git/week1` create a third file named `countries.txt` where you list three countries that you have visited (this does not need to be true, you can just write the names of three random countries), and commit your changes
+6. Add a fourth country name to the file `countries.txt`, and commit your changes again.
+7. Push your changes into your `hyf-homework` repository on github.
+
+
+Commands that  you will need:
+`git clone <repository_link>` - clone a github repository
+`git add <file_name>` - tell git to start tracking a file and to update what will be commited
+`git commit -m "commit_message"` - commit your changes
+`git push origin master` - push your changes into your github repository into the branch master.
+
+Useful git commands:
+ `git status`
+ `git log`
+ `git log --oneline`
+
+Command line commands that might be useful:
+`cd <folder_name>` - to go inside folder_name
+`cd ..`- to go to the parent folder of your current folder
+`mkdir <folder_name>` - to create a new folder
+`ls`- list all the folder contents
+`ls .*` - list all the folder contents, also invisible folders/files (e.g. the git folder)
+
+
```

Comment:
I'd add that this part of the homework is optional (assuming it's going to stay optional)